Disable cascading writes in ambiguous cases (#3794)
ElektraInitiative/libelektra: doc/tutorials/cascading.md

The **spec** namespace is used to store metadata about keys and therefore Elektra handles the **spec** namespace differently to other namespaces. The following part of the tutorial is dedicated to the impact of the **spec** namespace on cascading lookups.

## Write Operations and the cascading Namespace

> When a value is to be written to a cascading key, i.e. a key starting with '/', only cascading keys that resolve to some existing key may be used.

Reviewer: ```suggestion
If a value is to be written to a cascading key, i.e., a key starting with '/', only cascading keys that resolve to an existing key will be used.
```
